config/config.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+import (
+	"github.com/facebookgo/clock"
+)
+
+var (
+	Clock clock.Clock = clock.New()
```

Comment:
why not just add the clock to the `TaskConfig`?

Reply:
I went through this though process:
1. I added it to the struct
2. I wanted a way to default the value to clock.New()
3. I also wanted it to be a mandatory property since the function would fail if it defaults to `<nil>`
4. I added a constructor function to do that, which ends up taking a ton of parameters
5. I ended up having to update all the struct instantiations, so I gave up

I struggled with golang trying to get a simple way to add the default value to be `clock.New()`. If you have better ideas let me know.